modules/external-auth-providers.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * authentication/external-auth.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="external-auth-providers_{context}"]
+= Direct authentication identity providers
+
+Direct authentication has been tested with the following OpenID Connect (OIDC) identity providers:
+
+* Keycloak
+* Microsoft Entra ID
```

Comment:
@bergerhoffer , @anjaltelang , https://docs.redhat.com/en/documentation/openshift_container_platform/4.18/html/authentication_and_authorization/configuring-identity-providers#identity-provider-oidc-supported_configuring-oidc-identity-provider calls it "Microsoft identity platform (Azure Active Directory v2.0)". To be consistent, we can update https://docs.redhat.com/en/documentation/openshift_container_platform/4.18/html/authentication_and_authorization/configuring-identity-providers#identity-provider-oidc-supported_configuring-oidc-identity-provider with "Microsoft Entra ID"

Reply:
Definitely, will update